plugins/language_json.lua

```diff
@@ -0,0 +1,21 @@
+-- mod-version:3 priority:90
+
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "JSON",
+  files = { "%.json$" },
+  comment = nil,
+  patterns = {
+    { pattern = '\"[a-zA-Z0-9_ -]*\":', type = "keyword" }, -- key
+    { pattern = '\"[^\n]*"', type = "string" }, -- value
```

Comment:
I wonder if we could use the second pattern for the initial part of the first, like:
```suggestion
    { pattern = '"[^\n]*"%s*:', type = "keyword" }, -- key
    { pattern = '"[^\n]*"', type = "string" }, -- value
```
So that it would recognize keys of any kind.

Reply:
In the end the only issue is that it might break when trying to escape strings, but we can live with that for now.